### DISCUSSION TITLE
What is the TVEventHandler way of doing TVMenuControl.enableTVMenuKey

### DISCUSSION THREAD

**Discussion by oscadev** [react-native-tvos/react-native-tvos#361] [Q&A] [answered]
TVEventHandler.enableTVMenuKey() isnt a thing, so dont know how to achieve "dont exit the app on back button, except on home page". Trying to update a codebase I made like 3 years ago. Im using react native navigation native stack if that matters.

**Comment by douglowder** [comment, marked as answer]
This is mentioned in the `README.md` in this repo :)

- _TVEventControl_: (Formerly "TVMenuControl") (Apple TV only) This module provides methods to enable and disable firing of two types of events from the Apple TV Siri remote:
  - `enableTVMenuKey`/`disableTVMenuKey`:  Method to enable and disable the menu key gesture recognizer, in order to fix an issue with Apple's guidelines for menu key navigation (see https://github.com/facebook/react-native/issues/18930 ).  The `RNTester` app uses these methods to implement correct menu key behavior for back navigation.
  - `enableTVPanGesture`/`disableTVPanGesture`: Methods to enable and disable detection of finger touches that pan across the touch surface of the Siri remote. See `TVEventHandlerExample` in the `RNTester` app for a demo.

**Reply by oscadev** [reply]
Thanks for the reply. I had seen this already, but I dont know where "RNTester" is so that I can find the example of it being used. I can only find a repo named RNTester by another person and that hasnt been updated in 5 years

**Reply by douglowder** [reply]
It is `packages/rn-tester` in this repo
